tests/Unit/Controllers/DiagramControllerTest.php

```
@@ -841,4 +841,18 @@ public function testDiagramAiWorkflowOptionsAreTenantScoped(): void
         $this->assertSame(\StratFlow\Services\AiWorkflowContext::DIAGRAM_OUTPUT_TOKENS, $options['max_output_tokens']);
         $this->assertTrue($options['allow_openai_fallback']);
     }
+
+    public function testDiagramAiWorkflowOptionsCarryStableCustomerActionId(): void
+    {
+        $actionId = str_repeat('a', 32);
+        $method = new \ReflectionMethod(DiagramController::class, 'diagramAiOptions');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '848', column '23'). (undefined)

(MissingImport)